helm/action/create.go

```diff
@@ -1,33 +1,100 @@
 package action
 
 import (
+	"io/ioutil"
 	"os"
-	"path"
+	"path/filepath"
+	"text/template"
 )
 
 import (
 	"github.com/deis/helm/helm/log"
+	"github.com/deis/helm/helm/model"
 )
 
+// readmeSkel is the template for the README.md
+const readmeSkel = `# {{.Name}}
+
+Describe your chart here. Link to upstream repositories, Docker images or any
+external documentation.
+
+If your application requires any specific configuration like Secrets, you may
+include that information here.
+`
+
+// manifestSkel is an example manifest for a new chart
+const manifestSkel = `apiVersion: v1
+    kind: Pod
+    metadata:
+      name: example-pod
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: example
+      image: "alpine:3.2"
+        command: ["/bin/sleep","9000"]
+`
+
 // Create a chart
 //
 // - chartName being created
 // - homeDir is the helm home directory for the user
 func Create(chartName, homeDir string) {
 
-	skeletonDir := path.Join(homeDir, CachePath, "skel")
+	chart := newSkelChartfile(chartName)
+	chartDir := filepath.Join(homeDir, WorkspaceChartPath, chartName)
+
+	// create directories
+	if err := os.MkdirAll(filepath.Join(chartDir, "manifests"), 0755); err != nil {
+		log.Die("Could not create %q: %s", chartDir, err)
+	}
+
+	// create Chartfile.yaml
+	if err := chart.Save(filepath.Join(chartDir, "Chart.yaml")); err != nil {
+		log.Die("Could not create Chart.yaml: err", err)
+	}
+
+	// create README.md
+	if err := createReadme(chartDir, chart); err != nil {
+		log.Die("Could not create README.md: err", err)
+	}
 
-	if fi, err := os.Stat(skeletonDir); err != nil {
-		log.Die("Could not find %s: %s", skeletonDir, err)
-	} else if !fi.IsDir() {
-		log.Die("Malformed skeleton: %s: Must be a directory.", skeletonDir)
+	// create example-pod
+	if err := createExampleManifest(chartDir); err != nil {
+		log.Die("Could not create example manifest: err", err)
 	}
 
-	chartDir := path.Join(homeDir, WorkspaceChartPath, chartName)
+	log.Info("Created chart in %s", chartDir)
+}
+
+// newSkelChartfile populates a Chartfile model with example data
+func newSkelChartfile(chartName string) *model.Chartfile {
+	return &model.Chartfile{
+		Name:        chartName,
+		Home:        "http://example.com/your/project/home",
+		Version:     "0.0.1",
```

Comment:
let's change the version number to 0.1.0. According to semver, 0.1.0 is a minor release, whereas 0.0.1 is a patch release of 0.0.0 which doesn't make that much sense :)